src/platform/Infineon/CYW30739/CHIPPlatformConfig.h

```diff
@@ -23,7 +23,7 @@
 
 #pragma once
 
-#define CHIP_CONFIG_MAX_FABRICS 5
+#define CHIP_CONFIG_MAX_FABRICS 4
```

Comment:
Spec minimum to support is 5. You cannot pass cert with 4.

Reply:
I changed it back to 5.